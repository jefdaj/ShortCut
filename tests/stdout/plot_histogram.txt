[plot image '$TMPDIR/cache/bin/a6d2b88425.png',
 plot image '$TMPDIR/cache/bin/2ac58015c1.png',
 plot image '$TMPDIR/cache/bin/481723566d.png']
