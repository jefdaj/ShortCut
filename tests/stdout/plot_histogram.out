[plot image '$TMPDIR/cache/bin/c6c5a12ae1.png',
 plot image '$TMPDIR/cache/bin/4d5397648f.png',
 plot image '$TMPDIR/cache/bin/962f709576.png']
